com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/bundle/test/BundlerTest.java

```diff
@@ -101,8 +101,9 @@ public class BundlerTest {
     public static Collection<Platform[]> data() {
         List<Platform[]> data = new ArrayList<>();
 
-        String skipTest = System.getenv("DM_BOB_BUNDLERTEST_ONLY_HOST");
-        if (skipTest != null) {
+        String skipTest = System.getenv("DM_BOB_BUNDLERTEST_ONLY_HOST") != null ? System.getenv("DM_BOB_BUNDLERTEST_ONLY_HOST") : System.getProperty("DM_BOB_BUNDLERTEST_ONLY_HOST");
+
+        if (skipTest.equals("1")) {
```

Comment:
In general a boolean truth value is considered anything that isn't 0.
So would have suggested something like `if (!skipTest.equals("0"))`

However, I do think that the System.getProperty() returns null when it cannot find a value? So this code would crash in that scenario.

Reply:
I made fixes to make it clear and back compatable